templates/csharp/sso-tab-ssr/Components/Pages/Welcome.razor

```
@@ -54,11 +54,12 @@
             var user = await teamsUserCredential.GetUserInfoAsync();
             userName = user.DisplayName;
         }
+        loading = false;
```

Will `loading` be true when failed to get username?

---

No, if we want to report an error and end the loading, we need to add a try catch to solve the problem.